base/linux/fuzzos/recipes/pernosco_submit.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+set -e
+set -x
+
+# shellcheck source=base/linux/fuzzos/recipes/common.sh
+source "${0%/*}/common.sh"
+
+#### Install pernosco-submit
+
+apt-install-auto \
```

Comment:
This should use `sys-embed` instead of `apt-install-auto`. Packages installed with the latter could be uninstalled by `apt-get autoremove` after all recipes are run (in [cleanup.sh](https://github.com/MozillaSecurity/orion/blob/master/base/linux/fuzzos/setup.sh#L52)).

Reply:
Nice! I was not aware of `sys-embed` and `sys-update` (via below).